source/projectile/Projectile.hx

```diff
@@ -0,0 +1,33 @@
+package projectile;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxPoint;
+
+class Projectile extends FlxSprite {
+	private static var SPEED:Float = 0;
+
+	public function new() {
+		super(FlxG.width, FlxG.height);
+		kill();
+	}
+
+	override public function update(elapsed:Float) {
+		super.update(elapsed);
+	}
+
+	private function getSpeed() {
+		return SPEED;
+	}
```

Comment:
This is secretly dangerous because SPEED is static. If you don't override this `getSpeed()` method in each child class, it'll default to using this version of the function that uses `Projectile.SPEED`, which is 0

I was trying to integrate with it to test its behavior, and had quite a bit of trouble figuring out why bullets weren't moving anywhere.

The solution can be to make SPEED non-static (and have child classes choose their own value inside their constructor or something), use an instance variable instead of the `getSpeed()` method, or override `getSpeed` and make each class pick their own way of getting the value they want.